package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An app for helping mentors and mentees connect and schedule time to meet for coffee or any comparable beverage.",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "npm run eslint && npm run stylelint && jest",
```

Comment:
[optional] we probably can leave all of these as separate scripts. We'll set up CI using GitHub actions, and we can have different checks per tool. 
We could also leave them here until we actually setup Github actions.